### DISCUSSION TITLE
Someone stole my Mailbox...

### DISCUSSION THREAD

**Discussion by DomGibson** [dailydotdev/daily#1653] [Feature Requests]
I feel as though with Chrome, using this Daily dev which is now almost a companion to my day to day work, I am missing a mailbox. I always have the habit of going to the top-right to click gmail. This would be a great feature to choose which mailboxes to redirect to, one mailbox or multiple choice would be cool.

A small little icon that I can open my gmail or another button if I selected two mailboxes that links to my outlook.

**Comment by idoshamun** [comment]
Hi @DomGibson,

Have you tried our shortcuts feature?
